<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pid</class>
 <widget class="QWidget" name="Pid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bottom</string>
  </property>
  <widget class="QTabWidget" name="PID_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>8</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="POS">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;位置环PID参数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <attribute name="title">
     <string>POS</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="POS_PLabel">
        <property name="text">
         <string>P:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="POS_PSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="POS_ILabel">
        <property name="text">
         <string>I:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="POS_ISpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="POS_DLabel">
        <property name="text">
         <string>D:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="POS_DLabel_2">
        <property name="text">
         <string>Dead:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="limitAccLabel">
        <property name="text">
         <string>Limit Acc:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="maxAccLineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="POS_DSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="POS_DSSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="minPosLLabel">
        <property name="text">
         <string>Min POS_L</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="minPosLLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="minPosHLabel">
        <property name="text">
         <string>Min POS_H</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="minPosHLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="maxPosLLabel">
        <property name="text">
         <string>Max POS_L</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="maxPosLLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="maxPosHLabel">
        <property name="text">
         <string>Max POS_H</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="maxPosHLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="SPD">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;速度环PID参数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <attribute name="title">
     <string>SPD</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="SPD_PLabel">
        <property name="text">
         <string>P:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="SPD_PSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="SPD_ILabel">
        <property name="text">
         <string>I:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="SPD_ISpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="SPD_DLabel">
        <property name="text">
         <string>D:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="SPD_DSLabel">
        <property name="text">
         <string>Dead:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="limitSpdLabel">
        <property name="text">
         <string>Limit Spd:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="maxSpdLineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="SPD_DSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="SPD_DSSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="CUR">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;电流环PID参数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <attribute name="title">
     <string>CUR</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>155</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="CUR_PLabel">
        <property name="text">
         <string>P:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="CUR_PSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="CUR_ILabel">
        <property name="text">
         <string>I:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="CUR_ISpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="CUR_DLabel">
        <property name="text">
         <string>D:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="CUR_DSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="limitCurLabel">
        <property name="text">
         <string>Limit Cur:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="maxCurLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="adjustGroupComboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;切换PID调整组显示：1-低载PID参数，2-中载PID参数，3-重载PID参数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
